### PR TITLE
Contributing.md updates

### PR DESCRIPTION
**Pull request by davidkline-ms**
Overview
---
Update contributing.md to reflect the mrtk_development branch, supported platforms and documentation requirements (including a link to the mixed reality MicrosoftDocs repository.

- Fixes: #2016 .

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,31 +19,56 @@ If your change is more than a simple fix, please don't just create a big pull re
 Instead, start by [opening an issue](https://github.com/Microsoft/MixedRealityToolkit-Unity/issues) describing the problem you want to solve and how you plan to approach the problem. 
 This will let us have a brief discussion about the problem and, hopefully, identify some potential pitfalls before too much time is spent.
 
-Note:  If you wish to work on something that already exists on our backlog, you can use that work item as your proposal.  
+Note:  If you wish to work on something that already exists on our backlog, you can use that work item as your proposal.
+Please take ownership of the issue so that others do not duplicate your effort.
```
Comment (keveleigh) [inline CONTRIBUTING.md]: I believe only those with write access can [assign issues](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/).

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,31 +19,56 @@ If your change is more than a simple fix, please don't just create a big pull re
...
+9. Update the [documentation](#update-documentation) with additional information as needed.
+
+## Update Documentation
+
+The Mixed Reality Toolkit provides the following forms of documentation.
+
+- API
+This documentation is generated from the product code and is reviewed as part of **all** pull requests.
+- Conceptual
+Conceptual documentation is hosted on https://docs.microsoft.com/en-us/windows/mixed-reality. Please submit our changes via Pull Requuest at https://github.com/MicrosoftDocs/mixed-reality.
```
Comment (keveleigh) [inline CONTRIBUTING.md]: Requuest

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -18,31 +19,56 @@ If your change is more than a simple fix, please don't just create a big pull re
...
+- Readme.md files
+As part of your pull request, please update (or create) the Readme.md file in the appropriate feature folder. This will allow GitHub users to gain a high-level understanding of your new feature.
+
+
+## Supported Platforms
+
+The Mixed Reality Toolkit supports thge following mixed reality (AR/VR/XR) platforms.
```
Comment (keveleigh) [inline CONTRIBUTING.md]: thge

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -18,31 +19,56 @@ If your change is more than a simple fix, please don't just create a big pull re
...
+
+- Windows Mixed Reality
+    - Immersive devices
+    - Microsoft Hololens
+- Open VR
```
Comment (keveleigh) [inline CONTRIBUTING.md]: Nit: Open VR -> [OpenVR](https://docs.unity3d.com/Manual/VRDevices-OpenVR.html)

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -18,31 +19,56 @@ If your change is more than a simple fix, please don't just create a big pull re
...
+As part of your pull request, please update (or create) the Readme.md file in the appropriate feature folder. This will allow GitHub users to gain a high-level understanding of your new feature.
+
+
+## Supported Platforms
+
+The Mixed Reality Toolkit supports thge following mixed reality (AR/VR/XR) platforms.
+
+- Windows Mixed Reality
+    - Immersive devices
+    - Microsoft Hololens
```
Comment (keveleigh) [inline CONTRIBUTING.md]: HoloLens

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -18,31 +19,56 @@ If your change is more than a simple fix, please don't just create a big pull re
...
 A comment in the PR will let you know if you do.
 
 The project maintainers will review your changes. We aim to review all changes within three business days.
-Address any review comments, force push to your topic branch, and post a comment letting us know that there's new stuff to review.
+Address any review comments, push to your topic branch, and post a comment letting us know that there's new stuff to review.
 
 # Merge
 
 If the pull request review goes well, a project maintainer will merge your changes. Thank you for helping improve MixedRealityToolkit!
+
```
Comment (keveleigh) [inline CONTRIBUTING.md]: Empty spaces?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -18,31 +19,56 @@ If your change is more than a simple fix, please don't just create a big pull re
...
+The Mixed Reality Toolkit provides the following forms of documentation.
+
+- API
+This documentation is generated from the product code and is reviewed as part of **all** pull requests.
+- Conceptual
+Conceptual documentation is hosted on https://docs.microsoft.com/en-us/windows/mixed-reality. Please submit our changes via Pull Requuest at https://github.com/MicrosoftDocs/mixed-reality.
+- Readme.md files
+As part of your pull request, please update (or create) the Readme.md file in the appropriate feature folder. This will allow GitHub users to gain a high-level understanding of your new feature.
+
```
Comment (keveleigh) [inline CONTRIBUTING.md]: Extra line

File: CONTRIBUTING.md (hunk 1 above)
Comment (davidkline-ms) [reply CONTRIBUTING.md]: Fair enough. I will pull that line.

File: CONTRIBUTING.md (hunk 2 above)
Comment (davidkline-ms) [reply CONTRIBUTING.md]: grumble.... dumb sloppy typing :)

File: CONTRIBUTING.md (hunk 6 above)
Comment (davidkline-ms) [reply CONTRIBUTING.md]: ?

Comment (davidkline-ms) [reply CONTRIBUTING.md]: not sure where they came from... removing

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -5,8 +5,9 @@ If you have any questions, please reach out on the [HoloLens forums](https://for
 
 # Process
 
-1. [Make a proposal](https://github.com/Microsoft/MixedRealityToolkit-Unity/issues) (either new, or for one of the elements in our backlog)
-2. Implement the proposal and its tests.
+1. [Make a proposal](https://github.com/Microsoft/MixedRealityToolkit-Unity/issues)
+> If you are implementing something from our backlog, you do not need to file a new proposal.
```
Comment (StephenHodgson) [inline CONTRIBUTING.md]: maybe a bullet point instead?  It doesn't seem to fit right with the numbers, imo

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -18,31 +19,52 @@ If your change is more than a simple fix, please don't just create a big pull re
...
 5. Make sure that all the tests continue to pass.
 6. Follow the [Coding Guidelines](/CodingGuidelines.md).
 7. Ensure the code and feature(s) are documented as describred in the [Documentation Guidelines](/DocumentationGuidelines.md).
-8. Ensure the code is [WACK compliant](https://developer.microsoft.com/en-us/windows/develop/app-certification-kit). To do this, generate a Visual Studio solution, right click on project; Store -> Create App Packages. Follow the prompts and run WACK tests. Make sure they all succeed.
-9. Ensure you update the [README](README.md) with additional documentation as needed.
-10. Also update the [MixedRealityToolkit-Unity wiki](https://github.com/Microsoft/MixedRealityToolkit-Unity/wiki) if you think it will be useful for other developers.
+8. Ensure the code works as intended on all [platforms](#supported-platforms).
+> For Windows UWP projects, your code must be [WACK compliant](https://developer.microsoft.com/en-us/windows/develop/app-certification-kit). To do this, generate a Visual Studio solution, right click on project; Store -> Create App Packages. Follow the prompts and run WACK tests. Make sure they all succeed.
```
Comment (StephenHodgson) [inline CONTRIBUTING.md]: Same thing here.

File: CONTRIBUTING.md (hunk 8 above)
Comment (davidkline-ms) [reply CONTRIBUTING.md]: sure

**Pull request merged**